jsonrpc/txpool_endpoint.go

```diff
@@ -0,0 +1,115 @@
+package jsonrpc
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/minimal/types"
+)
+
+// Txpool is the txpool jsonrpc endpoint
+type Txpool struct {
+	d *Dispatcher
+}
+
+type ContentResponse struct {
+	Pending   map[types.Address]map[uint64]*transaction		`json:"pending"`
+	Queued 		map[types.Address]map[uint64]*transaction		`json:"queued"`
+}
+
+type InspectResponse struct {
+	Pending   map[string]map[string]string	`json:"pending"`
+	Queued		map[string]map[string]string	`json:"queued"`
+}
+
+type StatusResponse struct {
+	Pending   uint64	`json:"pending"`
+	Queued 		uint64	`json:"queued"`
+}
+
+/** 
+ * Content - implemented according to https://geth.ethereum.org/docs/rpc/ns-txpool#txpool_content
+ */
+func (t *Txpool) Content() (interface{}, error) {
+	pendingTxs, queuedTxs := t.d.store.GetTxs()
+	pendingRpcTxns := make(map[types.Address]map[uint64]*transaction)
+	for address, nonces := range pendingTxs {
+		pendingRpcTxns[address] = make(map[uint64]*transaction)
+		for nonce, tx := range nonces {
+			blockHash, _ := t.d.store.ReadTxLookup(tx.Hash)
+			block, _ := t.d.store.GetBlockByHash(blockHash, false)
+			// using 0 as txIndex
+			pendingRpcTxns[address][nonce] = toTransaction(tx, block, 0)
```

Comment:
`toTransaction` probably will throw panic because tx is not in any block at that time.